content/en/blog/2023/otel-dotnet-performance-impacts.md

```diff
@@ -0,0 +1,287 @@
+---
+title: Does OpenTelemetry in .NET Cause Performance Degradation?
+linkTitle: opentelemetry dotnet performance impacts
+date: 2023-03-10
+author: '[MartinDotNet](https://github.com/martinjt) (Honeycomb)'
+---
+
+Contrary to Betteridge’s Law of Tabloid Headlines, the answer is “yes,” but
```

Comment:
I don't know about this law, so please link to a wikipedia page or a similar relevant source explaining it.